temoa-uiuc/data_files/UIUC_2050_cost.sql

```diff
@@ -0,0 +1,3078 @@
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2026, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2027, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2028, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2029, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2030, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2031, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2032, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2033, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2034, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2035, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2036, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2037, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2038, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2039, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2040, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2041, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2042, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2043, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2044, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2045, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2046, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2047, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2048, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2049, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+INSERT INTO "Efficiency" VALUES('uiuc', 'GAS', 'ABBOTT', 2050, 'STM', 1.00, 'Converts steam to steam? Unsure.');
+--
+--INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2030, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
```

Comment:
This line i commented out is still in `11_uiuc.sql`, but it;s out of order and duplicate with another entry later on. Can @robfairh provide confirmation?

Reply:
You can remove that line in ```UIUC_2050_COST.sql```.

Your entries should be:

INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2000, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2021, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2022, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2023, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2024, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2025, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2026, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2027, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2028, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2029, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2030, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2031, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2032, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2033, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2034, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2035, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2036, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2037, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2038, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2039, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2040, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2041, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2042, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2043, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2044, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2045, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2046, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2047, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2048, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2049, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');
INSERT INTO "Efficiency" VALUES('uiuc', 'STM', 'TURBINE', 2050, 'ELC', 0.6, 'converts STM to ELC, CHP efficiency');